### PR TITLE
Fix issues with empty MOTDs - fallback to configured MOTD

### PR DESCRIPTION
**Pull request by onebeastchris**
If the description in ping info is empty, the following returns an empty array:
`String[] motd = MessageTranslator.convertMessageLenient(pingInfo.getDescription()).split("\n");`

That causes accessing `motd[0]` to throw an `ArrayIndexOutOfBoundsException` - this PR resolves that by falling back to the configured primary/secondary MOTD. Those also fall back to "Geyser" if empty.

Additionally, similarly to how spigot/paper pings allow hiding player counts, the same can be done by leaving the optional player list empty - in such cases, this ensures that we do not send out player names + bogus max/online player counts.

Fixes https://github.com/GeyserMC/Geyser/issues/4223

### CODE REVIEW

File: bootstrap/velocity/src/main/java/org/geysermc/geyser/platform/velocity/GeyserVelocityPingPassthrough.java
Comment (Konicai) [inline bootstrap/velocity/src/main/java/org/geysermc/geyser/platform/velocity/GeyserVelocityPingPassthrough.java:71]: instead of adding a whole new if-else block, use `Optional#map(...)` to convert it to an optional of the player counts, and then use `orElse(<0 or 1>)`

**Review by Konicai — CHANGES_REQUESTED** [review]

File: bootstrap/velocity/src/main/java/org/geysermc/geyser/platform/velocity/GeyserVelocityPingPassthrough.java
Comment (Konicai) [reply bootstrap/velocity/src/main/java/org/geysermc/geyser/platform/velocity/GeyserVelocityPingPassthrough.java:71]: like this https://github.com/GeyserMC/Geyser/pull/3930/files#diff-74e0ca9af638237203871557a54d9077a0d6b83877650880139161ff78c083c1R59

Comment (onebeastchris) [reply bootstrap/velocity/src/main/java/org/geysermc/geyser/platform/velocity/GeyserVelocityPingPassthrough.java:71]: Seeing how the other PR adds that fix anyways, i'll wait until it is merged and then rebase on that :p
Thanks for the link :)

**Review by Konicai — APPROVED** [review]

**Pull request merged**
